core/templates/dev/head/pages/skill-editor-page/questions-tab/questions-tab.directive.ts

```diff
@@ -33,6 +33,8 @@ require(
   'state-editor.service.ts');
 require('services/AlertsService.ts');
 require('services/contextual/UrlService.ts');
+require('services/QuestionsListService.ts');
```

Comment:
Switch line 35 and 36. We are ordering also by the depth of the folder structure where is the file located.

Reply:
Done.